ISISComputingGroup/EPICS-inst_servers
# Ticket1016: refactoring some of the many os joins

ISISComputingGroup/IBEX/issues/1016

To test:

* Run the unit tests
* Start up EPICS and the GUI then check you can load, save and edit configs and synoptics.
